go/teams/chain.go

```diff
@@ -1716,6 +1728,23 @@ func (t *teamSigchainPlayer) addInnerLink(mctx libkb.MetaContext,
 		if err != nil {
 			return res, err
 		}
+	case libkb.LinkTypeTeamBotSettings:
+		if err = enforce(LinkRules{
+			Admin:               TristateRequire,
```

Comment:
Admin should be optional. It's an optional reason that the poster is an admin that only exists because of implicit admins. Though it's probably present in all links, that's just laziness.